app/code/Magento/InventoryInStorePickup/Model/Order/IsFulfilled.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickup\Model\Order;
+
+use Magento\InventoryApi\Api\Data\SourceItemInterface;
+
+class IsFulfilled
```

Comment:
I think naming is not precise here.

As order would be fulfilled after a customer would grab ordered items from the store

Reply:
Okay. Google Translate says 'Fulfillable' is not an existing word, and I trust it :)
Renamed to `CanBeFulfilled`

Reply:
Let's quit using Google translate for serious purposes :)

But, keeping it serious, some popular dictionaries know this word and even it's opposite
https://en.oxforddictionaries.com/definition/fulfillable
https://www.merriam-webster.com/dictionary/unfulfillable

Reply:
Okay. It is named `IsFulfillable` now